### PR TITLE
Add silkscreen for all passive-fn for example [0402,0603...] , and add plus sign prop and silkscreen

### PR DESCRIPTION
**Pull request by anas-sarkez**

### CODE REVIEW

File: src/helpers/+_sign.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import type { PcbSilkscreenPath } from "@tscircuit/soup"
```
Comment (seveibar) [inline src/helpers/+_sign.ts:1]: Can we make this filename a bit safer for different operating systems, we could call it plus_sign

File: src/helpers/passive-fn.ts [hunk 2]
```diff
@@ -163,16 +164,65 @@ export const passive = (params: PassiveDef): AnySoupElement[] => {
 
   if (pw === undefined) throw new Error("could not infer pad width")
   if (ph === undefined) throw new Error("could not infer pad width")
+  // if (params.w && params.p && params.pw) {
+
+  const PcbSilkscreenPath: PcbSilkscreenPath[] = [
```
Comment (seveibar) [inline src/helpers/passive-fn.ts:169]: Use camelCase for variable names. PascalCase is conventionally used for interfaces, types or classes

File: tests/output/cap footprint.svg [hunk 3]
```diff
@@ -1,7 +1,16 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="800" height="600"><style>
               .pcb-board { fill: #000; }
+<<<<<<< HEAD
```
Comment (seveibar) [inline tests/output/cap footprint.svg:3]: This file has merge conflicts, along with some others

**Review by seveibar — CHANGES_REQUESTED** [review]
Most svgs are broken due to meege conflicts

File: tests/res.test.ts [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import test from "ava"
 import { getTestFixture } from "tests/fixtures/get-test-fixture"
 import { su } from "@tscircuit/soup-util"
+import { res } from "src/fn/res"
 
 test("res_imperial0402", async (t) => {
   const { fp, snapshotSoup } = await getTestFixture(t)
-  const soup = fp.string("res_imperial0402").soup()
+  const soup = fp.string("2512").soup()
```
Comment (seveibar) [inline tests/res.test.ts:9]: Why was this changed? Imperial0402

**Review by seveibar — APPROVED** [review]

File: tests/res.test.ts (hunk 4 above)
Comment (anas-sarkez) [reply tests/res.test.ts:9]: oops I missed when I was testing all passive fn

**Comment by seveibar** [comment]
reverting this merge because it broke the repo publishing, @anas-sarkez make sure you are completing passing so you don't break the repo

image
